examples/mysql-galera/mysql.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Pod
```

Comment:
Per the [best practices](https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/config-best-practices.md) (#15) you should avoid using any naked pods.

Per your comment about providing these in case a user wanted them, I'd prefer not to confuse users with the option (especially one not conforming to best practices). If someone really wanted to take this approach, they can always pull the template out of the ReplicationController template.